lib/unique.js

```diff
@@ -0,0 +1,24 @@
+module.exports = unqiue;
+
+var walk = require('./walk');
+
+function unqiue(deps) {
```

Comment:
ffs, I made that same mistake multiple times throughout the code. I'll fix before merging.